CardForm/src/main/java/com/braintreepayments/cardform/view/InitialValueCheckBox.java

```diff
@@ -0,0 +1,63 @@
+package com.braintreepayments.cardform.view;
+
+import android.content.Context;
+import android.os.Bundle;
+import android.os.Parcelable;
+import android.util.AttributeSet;
+
+import androidx.appcompat.widget.AppCompatCheckBox;
+
+/**
+ * Sets an initial CheckBox checked state that is overwritten when restoring this view.
+ */
+public class InitialValueCheckBox extends AppCompatCheckBox {
```

Comment:
Didn't we decide on toggle? https://braintree.slack.com/archives/GB6RZSQV9/p1555624166178500